lua/compe/completion/init.lua

```diff
@@ -222,5 +251,4 @@ function Completion.display(self, context)
   end)
 end
 
-return Completion
-
+return setmetatable({}, { __index = M })
```

Comment:
I don't love this `M` convention but I can permit use it.

Reply:
Yah, I have mixed feeling about it, specially when trying to search with ctags, so I only use it for small non important files. 
But when you merge completion/init.lua to compe/init.lua keep `compe` don't use M

Reply:
I worry about the `M` convention. Is it common practice in Lua...?

If it's common practice, I'm ok to use it.

OK, if I merge `completion/init.lua` with `compe/init.lua`, I use `compe` for the name.

Reply:
@hrsh7th `M` is common enough that makes sense to use it over `table`, `t`, `a`, `_`, ... etc. it actually stands for "module".

> OK, if I merge completion/init.lua with compe/init.lua, I use compe for the name.
exactly, the new file name should be `compe/init.lua`